index.js

```diff
@@ -2,38 +2,39 @@ var BUFFERED_ACTION_RETURN = 'redux-action-buffer: buffered action'
 
 var setImmediate = typeof global !== 'undefined' && typeof global.setImmediate !== 'undefined' ? global.setImmediate : setTimeout
 
-module.exports = function bufferActions (breaker, cb) {
+module.exports = function bufferActions(breaker, cb, passthrough) {
```

Comment:
how about this api instead:
`bufferActions(options, cb)`
where options is an object with `{ breaker, passthrough }`.

This way we can keep backwards compat via mapping the arguments if a string is provided in the first argument. Also keeps callback as final arg which is idiomatic.